tools/java/org/unicode/cldr/tool/EFixedDecimal.java

```diff
@@ -0,0 +1,82 @@
+package org.unicode.cldr.tool;
+
+import com.ibm.icu.text.PluralRules.FixedDecimal;
+import com.ibm.icu.text.PluralRules.IFixedDecimal;
+import com.ibm.icu.text.PluralRules.Operand;
+import com.ibm.icu.text.UnicodeSet;
+
```

Comment:
Introduced a new EFixedDecimal for generating exponent samples. Basically just adds the 'e' feature.